packages/synthetic-chain/upgrade-test-scripts/run_test.sh

```diff
@@ -6,12 +6,20 @@ set -e
 
 source ./env_setup.sh
 
-PROPOSAL_PATH=$1
+PROPOSAL=$1
+if [ -z "$PROPOSAL" ]; then
+    echo "Must specify what proposal to use"
+    exit 1
+fi
+
+echo "[$PROPOSAL] Starting agd"
+
+echo "[$PROPOSAL_PATH] Starting agd"
```

Comment:
Detritus?

```suggestion
```

Reply:
ah, probably bad rebase. will correct in next PR